tests/components/sensor/test_recorder.py

```diff
@@ -3742,69 +3742,62 @@ def _sum(seq, last_state, last_sum):
         "sensor.test4": None,
     }
     start = zero
-    with freeze_time(start) as freezer:
```

Comment:
This section is simply un-indented, as we can use the original freezer pytest fixture.